cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,43 @@
-describe('', () => {
+describe('CRUD article', () => {
+  let user;
+  let articleData;
   before(() => {
-
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      articleData = generateArticle;
+    });
+    cy.login(user.email, user.username, user.password);
   });
 
-  it('', () => {
-
+  it('should create the article', () => {
+    cy.visit('/editor');
+    cy.contains('.nav-link', 'New Article').click();
+    cy.get('[placeholder="Article Title"]')
+      .type(articleData.title);
+    cy.get('[placeholder="What\'s this article about?"]')
+      .type(articleData.description);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(articleData.body);
+    cy.contains('.btn', 'Publish Article')
+      .click();
+    cy.url().should('contain', 'article/');
```

Comment:
After creating the article, it would be a good idea to add some assertions to validate that the article was created correctly. For example, you could check that the new article's title and content are displayed correctly on the page.